microk8s-resources/actions/common/utils.sh

```diff
@@ -1372,6 +1374,11 @@ use_snap_env() {
   mkdir -p "${XDG_RUNTIME_DIR}"
 }
 
+run_host() {
```

Comment:
Wouldn't this break the snap confinement, as it assumes that the underlying host has the program available?

Reply:
indeed, this is only used on non-strict code paths